### ISSUE TITLE
[Bug] Calling Spawn() in init() can lead to sound effect loading failure used in spawned objects

### ISSUE THREAD

**Issue by YuLun-bili** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#507]
### Describe the bug

```lua
function init()
	print("init spawn")
	Spawn(GetString("spawn.builtin-vehiclepack.13.path"), Transform(Vec(8, 1, 0)))
	print("ends")
end

function tick()
	if not spawned then
		print("1st tick spawn")
		Spawn(GetString("spawn.builtin-vehiclepack.13.path"), Transform(Vec(15, 1, 0)))
		print("ends")
		spawned = true
	end
end
```
![image](https://github.com/Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker/assets/70589524/f182f7af-6284-4b1e-9883-aa6202a067aa)

### Steps to reproduce the behavior

clearly written in previous section

### Expected behavior

no sound effect loading failure

### Environment

- Teardown Version 1.5.3
- OS: Win 11

**Comment by Gregory-Gregory** [comment]
Hi @YuLun-bili,

Thank you for your bug report. We have forwarded this information to the QA team for further investigation.

**Comment by YuLun-bili** [comment]
already resolved in game (version 1.6.0), fix maybe deployed in 1.5.4
